src/main/java/io/strimzi/kafka/bridge/bridge/BridgeCommonConfig.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.kafka.bridge.bridge;
+
+import io.strimzi.kafka.bridge.config.AbstractConfig;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * Bridge related configuration
+ */
+public class BridgeCommonConfig extends AbstractConfig {
```

Comment:
Why having a new class for that, isn't `BridgeConfig` enough for containing this info.
It has the Kafka config (with producer and consumer) and `T` endpoint config (so AMQP or HTTP).
It could have info like `id` which are bridge wide.